reana-ui/src/pages/workflowList/components/WorkflowList.js

```diff
@@ -42,22 +45,32 @@ export default function WorkflowList({ workflows, refreshedAt }) {
           duration,
           completed,
           total,
+          size,
           status,
-        }) => (
+        } = workflow;
+        return (
           <div
             key={id}
             onClick={() => history.push(`/details/${id}`)}
             className={`${styles["workflow"]} ${
               status === "deleted" ? styles["deleted"] : ""
             }`}
           >
-            <div>
+            <div className={styles["details-box"]}>
```

Comment:
Somewhere we are following `dot` representation(`x.y`) and at other places, we are referring like `key` in the dict(`x["y"]`). am I missing something?

```suggestion
            <div className={styles.details-box}>
```

Reply:
Hyphens are not allowed in dot representation, that's why we have some mixture.. which I agree it's not very nice...

Reply:
okay, We can create another ticket to address this mixture to use only one of them. 👍